lib/ansible/playbook/base.py

```diff
@@ -112,6 +112,9 @@ def _generic_g(prop_name, self):
         if hasattr(self, method):
             return getattr(self, method)()
 
+        if prop_name not in self._attributes:
+            return None
+
```

Comment:
I'd like to see a comment either here or on the commit explaining this. 

It seems like a general bug fix and unrelated to py2/py3. If it is related to py2/py3 I'd like to see an explanation.